src/plot.rs

```diff
@@ -100,7 +87,6 @@ pub fn plot_total_energy(data: Vec<PlotDatum>, config: &Config) {
             },
         )
         .expect("Should be able to estimate the text size");
-
     let x_label_size = root_drawing_area
         .estimate_text_size(
             &format_label(&y_max_total_energy),
```

Comment:
This is an error. I'll fix it now.

Reply:
Actually, comparing before and after, I think it looks better using the y label's size for both. It looks like this only affects the distance between the label and the axis values.
Before:
![image](https://github.com/keplerplanetary/kepler_cli/assets/74881094/0f45d17c-49ad-4c79-8a72-75eef06e7c88)
After:
![image](https://github.com/keplerplanetary/kepler_cli/assets/74881094/98cd794d-f221-44dc-bcc9-c2e78da8ad45)
To me, it looks more even to use the same value for both x and y.

Reply:
The energy is indeed J (Joules).

Typically one would write the label as "Total Energy / J", because if we take the total energy, which is a number with the unit Joule, and divide by Joule, we are left with a number, which is then plotted or displayed on the coordinate axes.

Reply:
I think I will change time to `Time / s` as well.